dataactcore/config_example.yml

```diff
@@ -137,6 +137,7 @@ logging:
             - use_aws
             - local
             - aws_bucket
+            - certified_bucket
```

Comment:
Do we want to put this key in as an example above in the broker section as well?